doc/user_guide/advanced.rst

```diff
@@ -149,7 +149,7 @@ implemented in the ``Solver.warm_up`` method, which is empty by default and
 called after the `set_objective` method. For solvers with
 ``sampling_strategy`` in ``{'tolerance',  'iteration'}``, simply calling the
 ``Solver.run`` with a simple enough value is usually enough. For solvers with
-``sampling_strategy`` set to ``'callback'``, it is possible to call
+``sampling_strategy`` in ``{'callback', 'run_once'}``, it is possible to call
```

Comment:
for sampling strategy `run_once`, calling the  `run_once` method makes little sense no?

Reply:
Indeed